circle.yml

```diff
@@ -173,6 +173,9 @@ jobs:
     working_directory: ~/TheBrain2.0
     steps:
       - checkout
+      - run:
+          name: Generating branch name
+          command: BRANCH_NAME=$(echo ${CIRCLE_BRANCH,,} | md5sum | awk -v branch=$CIRCLE_BRANCH '{ print tolower(substr(branch,0,15))"-"substr($1,0,8) }') && echo "export BRANCH_NAME=$BRANCH_NAME" >> $BASH_ENV
```

Comment:
Env var `BRANCH_NAME` should also be named to something like `BUILD_VERSION_LABEL`, please rename all occurrences.